lisa/runners/lisa_runner.py

```diff
@@ -354,8 +354,16 @@ def _run_test_task(
                 f"because after test case '{test_result.name}', "
                 f"node(s) cannot be accessible."
             )
-            # check panic when node(s) in bad status
-            environment.nodes.check_kernel_panics()
+            try:
+                # check panic when node(s) in bad status
+                environment.nodes.check_kernel_panics()
+            except LisaException as identifier:
+                # not throw exception here, since it will cancel all tasks
+                # just print log here
+                self._log.debug(
```

Comment:
Besides log the failure, the environment should be marked as dirty, so it won't be reused.